public_html/wp-content/plugins/pattern-creator/src/style.css

```diff
@@ -1,54 +1,90 @@
 :root {
-	--grid-unit: 16px;
-	--border-color: #e0e0e0;
-	--sidebar-width: 280px;
+	--wp-admin-theme-color: #3858e9;
+	--wp-admin-theme-color-darker-10: #074fd2;
+	--wp-admin-theme-color-darker-20: #0646ba;
 }
 
-#block-pattern-creator {
-	position: relative;
-	height: 80vh;
-	margin-top: 2rem;
-	border: 1px solid var(--border-color);
+.interface-interface-skeleton {
+	left: 0;
 }
 
-#block-pattern-creator *,
-#block-pattern-creator *::before,
-#block-pattern-creator *::after {
-	box-sizing: inherit;
+/* Hide the admin bar. */
+body.is-fullscreen-mode #wpadminbar {
+	display: none;
 }
 
-#block-pattern-creator div {
-	outline: none;
+/* Hide the revisions UI. */
+.edit-post-last-revision__panel {
+	display: none;
 }
 
-#block-pattern-creator a {
-	color: #3858e9;
+/* Used for the dropdown btn */
+.interface-interface-skeleton__header {
+	z-index: 10000;
 }
 
-#block-pattern-creator fieldset {
-	border: 0;
-	padding: 0;
-	margin: 0;
+/* Update the main editing document color */
+.editor-styles-wrapper {
+	background: #2f2f2f;
+	font-family:
+		-apple-system,
+		BlinkMacSystemFont,
+		"Segoe UI",
+		Roboto,
+		Oxygen-Sans,
+		Ubuntu,
+		Cantarell,
+		"Helvetica Neue",
+		sans-serif;
 }
 
-#block-pattern-creator button,
-#block-pattern-creator input,
-#block-pattern-creator select,
-#block-pattern-creator textarea {
-	line-height: 1.2;
+.editor-post-title {
+	color: #fff;
 }
 
-html.interface-interface-skeleton__html-container {
-	position: static;
+.editor-post-title__input {
+	text-align: center;
+	font-weight: 400 !important;
 }
 
-.interface-interface-skeleton {
-	position: absolute;
-	height: unset;
-	left: 0;
-	top: 0;
+.block-editor-block-list__layout {
+	margin: 0 auto;
+	background: #fff;
+	box-shadow: 0 0 0 1rem #fff;
+}
+
+.block-editor-post-preview__dropdown {
+	display: none;
+}
+
+.main-dashboard-button,
+.main-dashboard-button:hover {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	background: #23282e;
+	color: #fff;
+	height: 60px;
+	width: 60px;
+	font-size: 16px;
+	text-decoration: none;
+}
+
+.main-dashboard-button:hover {
+	background: #1e1e1e;
+}
+
+.viewport-header-control > button {
+	color: var(--wp-admin-theme-color);
+}
+
+/* Override gutenberg component style */
+.viewport-header-control .components-menu-group__label {
+	color: #2f2f2f;
 }
 
-.interface-interface-skeleton__footer {
-	position: absolute;
+.viewport-header-control__copy {
+	font-size: 10px;
```

Comment:
This is _very small_ in practice. What do you think, @shaunandrews?

<img width="291" alt="Screen Shot 2021-06-01 at 5 39 58 PM" src="https://user-images.githubusercontent.com/541093/120393967-cc4dd000-c300-11eb-9c64-afa457e6fdd5.png">

(I don't think changing this needs to block merge)

Reply:
Also noting that since this uses `@wordpress/components/UnitControl`, "Viewport Width" label is not 'hideable' like other input labels are for Gutenberg input components.

Reply:
Since this is value can't have units, could we switch to a NumberControl or TextControl? would that let you hide the label?

Reply:
We can probably follow the lead from the Tools menu, which uses a 13px font-size.

<img width="315" alt="image" src="https://user-images.githubusercontent.com/191598/120546902-67f14600-c3be-11eb-91d5-3f7814959382.png">

Reply:
Moved to `px` and manually hid the label in 2d3c6dd.